docker/superset/templates/docker-entrypoint.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2022 Google LLC.
+#
+# This program is free software; you can redistribute it and/or modify
```

Comment:
GPL entrypoint? Isn't the solution apache licensed?

Reply:
done